frontend/packages/dev-console/integration-tests/features/web-terminal/web-Terminal-adminuser.feature

```diff
@@ -0,0 +1,28 @@
+Feature: Web Terminal for Developer user
+    As a user with admin rights, I should be able to use web terminal
+
+
+Background:
+    Given user has logged in as admin user
+    And user is at developer perspecitve
+    And user has installed Web Terminal operator
+    And user has created Web Terminal instance
+    And user has created namespace "aut-terminal-admin"
+    And user is at the Topolgy page
+
+
+@regression, @smoke
+Scenario: Create new project and use Web Terminal
+    Given user is at topology page
```

Comment:
Does the scenario need to start on a particular page?
The starting page here is irrelevant to the test.

Reply:
It's not mandatory but we have to start somewhere. That's why I have to add that statement.

Reply:
Please remove topology page from given statement for all scenarios, as this web terminal is available on master head